docs/admin/user-management.rst

```diff
@@ -75,23 +205,30 @@ statement returns an error::
     RoleAlreadyExistsException[Role 'Custom User' already exists]
 
 
+.. hide:
+
+    cr> DROP USER "Custom User";
+    DROP OK, 1 row affected (... sec)
+
+
 ``ALTER USER``
 ==============
 
 To alter the password for an existing user from the CrateDB database cluster use
-the :ref:`ref-alter-user` SQL statement::
+the :ref:`ref-alter-role` or :ref:`ref-alter-user` SQL statements::
 
     cr> ALTER USER user_a SET (password = 'pass');
     ALTER OK, 1 row affected (... sec)
 
 The password can be reset (cleared) if specified as ``NULL``::
 
-    cr> ALTER USER user_a SET (password = NULL);
+    cr> ALTER ROLE user_a SET (password = NULL);
     ALTER OK, 1 row affected (... sec)
 
 .. NOTE::
 
-    The built-in superuser ``crate`` has no password and it is not possible to set a new password for this user.
+    The built-in superuser ``crate`` has no password and it is not possible to
+    set a new password for this user.
```

Comment:
I reverted the addition, since this was only a formatting change on this PR.
So if you manage (or not) to login with `crate` user using client certificate please open a PR to update this sentence accordingly (trust, hba, and  possibly client cert).
Thank you!!